lib/src/pages/home_page.dart

```diff
@@ -119,10 +119,16 @@ class _HomePageState extends State<HomePage> {
             children: <Widget>[
               _categoryCard("Chemist & Drugist", "350 + Stores",
                   color: LightColor.green, lightColor: LightColor.lightGreen),
-              _categoryCard("Covid - 19 Specilist", "899 Doctors",
+              _categoryCard("Covid - 19 Specialist", "899 Doctors",
                   color: LightColor.skyBlue, lightColor: LightColor.lightBlue),
-              _categoryCard("Cardiologists Specilist", "500 + Doctors",
-                  color: LightColor.orange, lightColor: LightColor.lightOrange)
+              _categoryCard("Cardiologists Specialist", "500 + Doctors",
+                  color: LightColor.orange, lightColor: LightColor.lightOrange),
+              _categoryCard("Dermatologist", "300 + Doctors",
```

Comment:
@sathyapriyaa-sketch The color combination of primary and light color on Dermatologists and General Surgeon card is looking a bit odd. Due to which both cards have separate color shadows than their primary color. So my suggestion is to change these colors so that primary and light colors of the card should look similar just like first three cards.